File: testresources/csharp/benchmark/Domain/Direct/Violating/DeclarationReturnType_GenericType_OneTypeParameter.cs
namespace Domain.Direct.Violating
{
	using Domain.Direct;
	using System.Collections.Generic;
	using Technology.Direct.Dao;

	public class DeclarationReturnType_GenericType_OneTypeParameter : Base
	{
		public List<ProfileDAO> testUsageOfInstanceVariableDefinedByGenericTypeOneParameterType()
		{
		return profileDAOs;
		}
	}
}

File: src/test/resources/java/accuracy/src/domain/direct/violating/DeclarationReturnType_GenericType_OneTypeParameter.java
package domain.direct.violating;

import java.util.ArrayList;

import domain.direct.Base;
import technology.direct.dao.ProfileDAO;

public class DeclarationReturnType_GenericType_OneTypeParameter extends Base{

	public ArrayList<ProfileDAO> testUsageOfInstanceVariableDefinedByGenericTypeOneParameterType(){
		return profileDAOs;
	}
}